lib/hexpm/accounts/pwned/hexpm.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Hexpm.Accounts.Pwned.Hexpm do
+  @behaviour Hexpm.Accounts.Pwned
+
+  @base_url "https://api.pwnedpasswords.com/"
+  @weakness_threshold 1
+  @timeout 500
+
+  @spec password_breached?(String.t()) :: boolean
+  def password_breached?(string_password) do
+    string_password
+    |> hash_password()
+    |> occurrences_of_hash()
+    |> Kernel.>=(@weakness_threshold)
+  end
+
+  defp hash_password(string_password) do
+    :sha
+    |> :crypto.hash(string_password)
+    |> Base.encode16()
+    |> String.upcase()
```

Comment:
By default `Base.encode16` uses the upcase alphabet already, there's no need to upcase it afterwords again.